eiffel-syntax-and-usage/glossary.md

```diff
@@ -84,38 +95,56 @@ An Eiffel _domain_ is an infrastructure topological concept, which may or may no
+
 More about what an event is supposed to be can be read in the [event design guidelines](./event-design-guidelines.md).
 
 ### Link
+
 A _link_ is a directed connection between two [events](#event). With linked events you will get a directed acyclic graph of actions performed in your CI/CD [pipeline](#pipeline). Links are also semantic, as each link has a type describing the relationship between the link source and its target. Having linked events enables generic data aggregations and visualizations to be implemented. Together with the timestamps provided in each event, the directed semantic links enable the possibility to create generic pipeline measurements and other analytics compilations.
 
 More about links can be read in the [links object documentation](./the-links-object.md).
 
 ### Source Change
+
 A _source change_ is the unit of a review. It results in a single commit when merged to the Git repository.
 
 #### Examples of events related to source changes:
+
 - [EiffelSourceChangeCreatedEvent](../eiffel-vocabulary/EiffelSourceChangeCreatedEvent.md)
 - [EiffelSourceChangeSubmittedEvent](../eiffel-vocabulary/EiffelSourceChangeSubmittedEvent.md)
 
 ### Submit
+
 A _submit_ is the action of merging a [source change](#source-change) to its intended target branch.
 
 #### Examples of events related to submits:
+
 - [EiffelSourceChangeSubmittedEvent](../eiffel-vocabulary/EiffelSourceChangeSubmittedEvent.md)
 
 ## Terms Used in Documentation
+
 The terms below are not used in any events in the Eiffel protocol. They are added here to enable a homogenous and easy read of the protocol documentation. These terms are subject to change without notice if a better name is found or defined elsewhere. Their names should reflect the most commonly used terms elsewhere in the industry.
 
+### Occurrence
+
+An _occurrence_ is the capture of a statement of fact during the execution of a CI/CD [pipeline](#pipeline). This might occur because of a [source change](#source-change) being created/updated, an [artifact](#artifact) being built, an upstream artifact being delivered, or any other activity relevant to a CI/CD pipeline.
+
 ### Pipeline
-A _pipeline_ is an ordered set of [activities](#activity) often triggered by a [source change](#source-change) being created or [submitted](#submit).
+
+A _pipeline_ is an ordered set of [activities](#activity) often triggered by a [source change](#source-change) being created or [submitted](#submit). There exists no EiffelPipeline\*Events, as the pipeline is to be considered an activity in itself. A pipeline execution could thus be notified using EiffelActivity\*Events, and its included sub activities would then have an hierarchical connection to the pipeline activity.
```

Comment:
Again, "notified" probably isn't the right word. In this case I'd say "expressed with".

Reply:
Agreed